### DISCUSSION TITLE
AsciiDoc / adoc preview in mobile app

### DISCUSSION THREAD

**Discussion by damian-krawczyk** [community/community#12315] [Mobile]
Hi,
When I select markdown file from my repository I see two options:
- "View raw"
- "View as HTML"

When I select asciidoc file default view for them is "raw" and there is no option "View as HTML".

When I create README.adoc - HTML view is rendered pretty well like for README.md 👍🏻

**Can you add the same option for adoc files as you have for md files?**
- View as raw
- View as HTML (this as default view)

Greetings,
Damian Krawczyk

**Comment by eliperkins** [comment]
Thanks for this feedback, @damian-krawczyk! I think this sounds more like a bug than a feature, given that AsciiDoc is rendered appropriately in some contexts, but not others.

I've filed an issue in our internal repo to track this, and let you know what our team can do to address this.

**Reply by damian-krawczyk** [reply]
Hi @eliperkins are there any chances for this feature in nearest future? 🤩

**Reply by timunie** [reply]
I also vote for this. Ascii docs are really convinent but they are only rendered on the main landing page. If the ReadMe.adoc is in any sun dir, it's not rendered as HTML.

**Reply by damian-krawczyk** [reply]
@eliperkins Any chance of having this?

**Reply by turkerozturk** [reply]
On my android 8.1.0 mobile phone using Github application, I also can see only the raw adoc contents. They are colorized, but there is no preview. 2024-09-06

**Reply by buster** [reply]
Same here. Current android app.

**Reply by YAS-K** [reply]
Is there any news about this?

**Comment by MrNiSso** [comment]
🔴Can u ratings my app and give me to /10🌟

https://play.google.com/store/apps/details?id=com.fd.food_recipe0
